config/nginx/recordexpunge-nginx.staging.conf

```diff
@@ -0,0 +1,18 @@
+server {
+    listen $PORT default_server;
+    charset c.utf-8; #matches what is defined for the flask service env
+
```

Comment:
Please put a redirect block for https. I believe the code is as follows:

```
    if ($scheme = http) {
        return 301 https://$server_name$request_uri;
    }
```

Reply:
It is not, but this shit shall be solved.